about.html

```diff
@@ -0,0 +1,76 @@
+                </article>
+                </div>
+                <div id="">
+                <h3>Team Members</h3>
+                        <h5>Garnet</h5>
+                            <p>Original founder of FABL and concerned mother of the famous Ruby a.k.a “Red Riding Hood”. Garnet created FABL as a mother first, making sure that her products are catered towards the parents looking for something to keep their child safe and sound.</p>
+
+                        <h5>Coral</h5>
+                            <p>Co-Founder of FABL, and Father of Ruby. Coral saw the concern of his wife along with other parents and collaborated with Garnet to create a “kids first” app development company. He shares the passion of his wife to be certain that his and every other parentʼs child are taken care of to the fullest.</p>
+
+                        <h5>Laura</h5>
+                            <p>Along with our senior developer, Lauraʼs mission as Director of Development is to develop the best possible solutions to the common concerns of parents. Leading our senior developer to help create the perfect app for each scenario is the top priority for our team and company as a whole.</p>
+
+                        <h5>Taylor</h5>
+                            <p>As our Director of Marketing, Taylor is making sure that our products are easily accessible to the parents looking for aid in their childʼs safety. Having our applications visible to all walks of life makes sure that no parent is left trying to dig for crucial information only to be left empty handed. Taylor is making sure we can help as many parents and children as possible.</p>
+
+                        <h5>Millian</h5>
+                            <p>Working alongside the Director of Development, Millian is our senior developer working hard on the ins and outs of how our applications function and what they are able to do for our users. Making sure that parents receive every feature necessary to make sure they are comfortable is Millianʼs primary goal.</p>
+
+                        <h5>Haden</h5>
+                            <p>Having our applications fun and easy to use is crucial to us to make sure we give our users the best experience possible. Our Senior UIX Designer Haden is working day by day with feedback from our users to refine our products to fit the wants and needs of parents and their children alike.</p>
+                </div>
+            </section>
+            <section id="#"></section>
+            <section id="#"></section>
+            <nav>
```

Comment:
For this nav sectiong, you can use the back to top, to sections here OR you can delete this portion if you wish. Up to you.